### DISCUSSION TITLE
jpeg vs heic quality and filesize comparison

### DISCUSSION THREAD

**Discussion by DavidCWGA** [ImageMagick/ImageMagick#6795] [General]
Is there somewhere I can read about how "-quality" compares for jpeg and heic? heic is supposed to be more efficient for the same quality, but making a jpeg and a heic, both with "-quality 75" (for example), the heic file is considerably larger, which is the opposite of what I would expect.

Are expected results documented somewhere?

**Comment by snibgo** [comment]
The `-quality` number for JPEG and HEIC have different calibrations, so the same number used for the two formats will make files that are different sizes and different qualities. We might measure quality as, for example, the RMSE from a source image.

https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format claims:
> A HEIC photo takes up about half the space of an equivalent quality JPEG file.
